README.md

```diff
@@ -11,7 +11,7 @@ Tested with the following apps:
 * [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) (proprietary)
 * [KeePassXC (Linux, Windows, macOS)](https://keepassxc.org/) (open-source) Available via [download](https://keepassxc.org/download/), package repositories or [GitHub](http://www.github.com/keepassxreboot/keepassxc/) (Keepass also provides a plugin and Keepass2Android allows to use TOTP token)
 * [SailOTP (SailfishOS)](https://github.com/seiichiro0185/sailotp) (open source) Available via JollaStore or [Openrepos.net](https://openrepos.net/content/seiichiro0185/sailotp)
-
+* [OTP Auth](https://cooperrs.de/otpauth.html) (proprietary) Availabe via [Apple's App Store](https://itunes.apple.com/us/app/otp-auth/id659877384)
```

Comment:
could you please move the line up by one so that there is no blank line to the other list items but one below?

Reply:
There is no blank line as seen in this screen shot
![image](https://user-images.githubusercontent.com/4145112/54602156-e09e9e80-4a0e-11e9-9d7c-a53a2ecd5488.png)

Reply:
Take a look at the source. It does render correctly, but looks weird there.